src/games/strategy/triplea/settings/SettingInputComponentFactory.java

```diff
@@ -134,14 +136,14 @@ private static JPanel createRadioButtonPanel(
 
   }
 
-  private static <Type extends HasDefaults> SettingInputComponent build(
+  private static <T extends HasDefaults> SettingInputComponent<T> build(
```

Comment:
Note, merging anyways, Type is not a good name to start  with, not losing that much going to 'T' even though we should have a name here instead of 'T'